java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestTensorsInfo.java

```diff
@@ -193,7 +193,11 @@ public void testAddUnknownType_n() {
     @Test
     public void testAddInvalidRank_n() {
         try {
-            mInfo.addTensorInfo(NNStreamer.TensorType.INT32, new int[]{2,2,2,2,2});
+            int[] invalid_rank_dim = new int[NNStreamer.TENSOR_RANK_LIMIT + 1];
+            for (int i = 0; i < invalid_rank_dim.length; i++) {
+                invalid_rank_dim[i] = i;
```

Comment:
set i + 1 to avoid 0 dimension value.

Reply:
Thanks! I changed the line as `= i + 1;`